Makefile

```diff
@@ -1,5 +1,6 @@
 SHELL = bash
 
+REGISTRY ?= deislabs
```

Comment:
Oops! Thanks for fixing. Would you mind changing the default to `$(USER)` instead? That way when I run `make test-cli` I don't accidentally push to deislabs. 😊 

The CI build already sets REGISTRY to deislabs explicitly so it shouldn't hurt it.

Reply:
Definitely!  Pushed up amended commit.